tests/client.py

```diff
@@ -950,6 +990,27 @@ def create_deals(self):
         records = [response]
         return records
 
+    def create_tickets(self):
+        """
+        HubSpot API https://developers.hubspot.com/docs/api/crm/tickets
+        """
+        url = f"{BASE_URL}/crm/v4/objects/tickets"
+        record_uuid = str(uuid.uuid4()).replace('-', '')
+        data = {
+        "properties": {
+            "content": f"Created for testing purpose - {record_uuid}",
+            "hs_pipeline": "0",
+            "hs_pipeline_stage": "1",
+            "hs_ticket_priority": "MEDIUM",
+            "subject": f"Sample ticket name - {record_uuid}"
+        }
+        }
+
+        # generate a record
+        response = self.post(url, data)
+        records = [response]
+        return records
```

Comment:
```suggestion
        return [response]
```
please remove the inline variable which is immediately returned

Reply:
done